core/server/pom.xml

```diff
@@ -162,6 +178,50 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.qmino</groupId>
+        <artifactId>miredot-plugin</artifactId>
+        <version>1.6.2</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>restdoc</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <license>
+            UHJvamVjdHxvcmcuYWxsdXhpby5hbGx1eGlvLWNvcmUtc2VydmVyfDIwMTctMDItMjV8dHJ1ZSNNQzBDRlFDUm1SMXpVb3dMOUkyNXZrcGtJOVlmOHY3M2tnSVVIa0NxSFgvdUlYZmd3SHdSa1hFYUFMRDQ3Q0E9
```

Comment:
done ... fyi, I contacted the miredot guys to make sure it is okay to use their plugin in the open source project and they offered to provide us with a commercial version of the plugin for the purpose of generating the REST API ... I will update the license accordingly one I receive it from them